# 4.	Let X ~ N(100, 202). Find two values, a and b, symmetric about the mean, such that the probability of the random variable taking a value between them is 0.99. 

A.	90.5, 105.9 
B.	80.2, 119.8 
C.	22, 78 
D.	48.5, 151.5 
E.	90.1, 109.9


In [1]:
import scipy.stats as stats

In [2]:
# Given parameters
mean = 100
std_deviation = 20
probability = 0.99

In [4]:
# Calculate the values of a and b

In [3]:
a = stats.norm.ppf((1 - probability) / 2, loc=mean, scale=std_deviation)
print("a:", a)

a: 48.48341392902199


In [5]:
b = stats.norm.ppf(1 - (1 - probability) / 2, loc=mean, scale=std_deviation)
print("b:", b)

b: 151.516586070978


# so ,the values a and b that give a 0.99 probability of the random variable taking a value between them are approximately 48.48 and 151.52.